CHANGELOG.md

```diff
@@ -1,49 +1,72 @@
+- Add support for Stream-Stream and Table-Table right join ([#8845](https://github.com/confluentinc/ksql/pull/8845)) ([c8975d3](https://github.com/confluentinc/ksql/commit/c8975d36dc8a90870f67101e50e70dbc715fa31e))
+- clean up extra runtimes so the assignment is more focused ([#8935](https://github.com/confluentinc/ksql/pull/8935)) ([247e6a5](https://github.com/confluentinc/ksql/commit/247e6a52d041d3a1991ec5c2a98cdba0ab2edf53))
+- Support MIN/MAX udafs for Time/TS/Date types ([#8924](https://github.com/confluentinc/ksql/pull/8924)) ([6399d30](https://github.com/confluentinc/ksql/commit/6399d30f93320b7d196bad03fca11e825dbc23b4))
+
+### Bug Fixes
+
+- INSERT fails when serializing Proto/Avro nested Structs ([#9038](https://github.com/confluentinc/ksql/pull/9038)) ([cd6cbf2](https://github.com/confluentinc/ksql/commit/cd6cbf2df3eb27be624c21d436e0fdf80030d3c9))
+- INSERT/VALUES on a stream with SCHEMA_ID/SCHEMA_FULL_NAME fails ([#9047](https://github.com/confluentinc/ksql/pull/9047)) ([bfd910b](https://github.com/confluentinc/ksql/commit/bfd910b64fe71c12fd03ed9064571c1cb83a0c4d))
+- update QTT tests for Proto multiple schema definitions ([8e077cb](https://github.com/confluentinc/ksql/commit/8e077cb473f40e174004e1d443d9c42b844f6954))
+- Create stream fails when multiple Protobuf schema definitions exist ([#8933](https://github.com/confluentinc/ksql/pull/8933)) ([87c0ce1](https://github.com/confluentinc/ksql/commit/87c0ce1858b79e918f8b6fe316bb7f2759f574d4))
+- Guard null struct dereferncing inside function calls ([#8918](https://github.com/confluentinc/ksql/pull/8918)) ([51753aa](https://github.com/confluentinc/ksql/commit/51753aa19ee8038c022413ed57a782bc6d86c244))
+- INSERT VALUES fail when SR schema has a non-default name ([#8984](https://github.com/confluentinc/ksql/pull/8984)) ([75da9fd](https://github.com/confluentinc/ksql/commit/75da9fd1e7dc9eae47203f3e40f167de7e3a6526))
+- remove double quotes from json_records function ([#9028](https://github.com/confluentinc/ksql/pull/9028)) ([86544ca](https://github.com/confluentinc/ksql/commit/86544ca9d121ffa570b4a8c5fdf50f053cd9724e))
+- only make shared runtime that its chosen ([#8932](https://github.com/confluentinc/ksql/pull/8932)) ([eb215c8](https://github.com/confluentinc/ksql/commit/eb215c8d1209a9d59bbd9185c5499a88e686f5f6))
+- **docs:** functional tests min version for lower bound ([#8922](https://github.com/confluentinc/ksql/pull/8922)) ([c33ee7b](https://github.com/confluentinc/ksql/commit/c33ee7b892ace3b62886800ea9383e8fc0f0fa59))
+- back out post-3.1 changes to fix 7.1.x build ([#8599](https://github.com/confluentinc/ksql/pull/8599)) ([f42225d](https://github.com/confluentinc/ksql/commit/f42225d2061af37ba0edf04be94d1e6eebd061b9))
+
+### Reverts
+
+- Revert "fix: back out post-3.1 changes to fix 7.1.x build (#8599)" ([6c3320c](https://github.com/confluentinc/ksql/commit/6c3320c670a86a380a46dfe4c7208fdaf1f9159c)), closes [#8599](https://github.com/confluentinc/ksql/issues/8599)

```

Comment:
Does it make sense to remove the revert and the bug fix above that goes with it?  (I'm not reading too closely...)